client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DomainPicker from '@automattic/domain-picker';
```

Comment:
Nit: Can we move all those `@automattic/` to the `External dependencies` section above? While they're internal for the repo, they're actually external for the package we're using them in (`client/package.json` AKA `calypso`).

Reply:
Fixed by 0aa0786